<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>247</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>6</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_2" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item alignment="Qt::AlignHCenter">
                   <widget class="QRadioButton" name="video_radioButton">
                    <property name="font">
                     <font>
                      <family>Unispace</family>
                      <pointsize>9</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Video</string>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignHCenter">
                   <widget class="QRadioButton" name="audio_radioButton">
                    <property name="font">
                     <font>
                      <family>Unispace</family>
                      <pointsize>9</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>Audio</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QRadioButton" name="playlist_radioButton">
                 <property name="font">
                  <font>
                   <family>Unispace</family>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="tabletTracking">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Playlist</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="youtubeURL_Label">
                 <property name="font">
                  <font>
                   <family>Unispace</family>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>YouTube URL :</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="youtubeURL_lineEdit">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>25</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="changeQuality_pushButton">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="feather/gui_resources.qrc">
                   <normaloff>:/icons/tool.svg</normaloff>:/icons/tool.svg</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="downloadNow_pushButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="font">
                  <font>
                   <family>Unispace</family>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Download Now</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="addToQueue_pushButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="font">
                  <font>
                   <family>Unispace</family>
                   <pointsize>12</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Add to Queue</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>246</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="downloadProcess_scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>863</width>
           <height>435</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTableWidget" name="downloadProcess_tableWidget">
            <property name="font">
             <font>
              <family>Unispace</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::IgnoreAction</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideLeft</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>173</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>70</number>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Thumbnail</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Channel</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Title</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Progress</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="feather/gui_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>changeQuality_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>download_options_popup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadNow_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>download_now()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addToQueue_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>add_to_queue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audio_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>enable_download_and_linkEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>video_radioButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>enable_download_and_linkEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>download_options_popup()</slot>
  <slot>download_now()</slot>
  <slot>add_to_queue()</slot>
  <slot>enable_download_and_linkEdit()</slot>
 </slots>
</ui>
